.gitignore

```diff
@@ -4,3 +4,4 @@ jsconfig.json
 npm-debug.log
 .vscode
 .DS_Store
+.eslintcache
```

Comment:
This is to allow people to use `eslint --cache`, which doesn't lint unchanged files.